gateway/1.4/modules/ROOT/pages/flex-conn-manage.adoc

```diff
@@ -13,18 +13,19 @@ Get started using Flex Gateway in Connected Mode with the following tasks:
 single replica. Running multiple replicas enables scaling, load balancing,
 and high availability.
-* xref:flex-conn-tls-config.adoc[Configure TLS Context] - 
-include::partial$task-tls-config.adoc[tags=intro1]
+* xref:flex-conn-tls-config.adoc[Configure TLS Context] - Configure a TLS Context to support regular TLS and mTLS in both inbound and outbound directions.
 * xref:flex-conn-message-log.adoc[Configure Message Logging] - Applying a Message Logging policy
 to your API instance in API Manager allows you to view basic logs for APIs managed by Flex Gateway
 within API Manager.
 * xref:flex-conn-third-party-logs-config.adoc[Configure Flex Gateway Logs for Third-Party Services] - Anypoint Flex Gateway enables you to configure log output for consumption by third-party services such as Splunk.
 * xref:flex-conn-shared-storage-config.adoc[Configure Shared Storage] - Anypoint Flex Gateway enables you to configure shared storage for distributed caching and rate limiting policies.
-* xref:api-manager::create-instance-task.adoc[Add an API Instance in API Manager] - An API Instance is an instantiation of an API that describes the configuration of the deployed API. The API can be deployed on one of the following three runtime options: Flex Gateway, Mule Gateway, or Service Mesh. 
+* xref:api-manager::create-instance-task.adoc[Add an API Instance in API Manager] - An API Instance is an instantiation of an API that describes the configuration of the deployed API.
 * xref:api-manager::configure-api-task.adoc[Configure an API Instance Endpoint in API Manager] - Depending on the runtime and API type, you can
 use several optional configurations to customize your API Instance to fit your specific situation.
 * xref:policies::policies-included-apply.adoc[Apply a Policy in API Manager] - A policy extends the functionality of an API and enforces certain capabilities such as security. A policy can control access and traffic. For example, a policy can generally control authentication, access,
 allotted consumption, and service level access (SLA).
 * xref:flex-use-api-alerts.adoc[Use API Alerts] - Using API alerts for Flex Gateway in Connected Mode, you can add and configure email
 notifications that flag unusual API behavior.
-* xref:flex-view-api-metrics.adoc[View Key Metrics for Flex Gateway APIs] - API Manager and Anypoint Monitoring collect metrics on active Flex Gateway APIs. You can view charts related to these metrics and filter the data by date.
\ No newline at end of file
+* xref:flex-view-api-metrics.adoc[View Key Metrics for Flex Gateway APIs] - API Manager and Anypoint Monitoring collect metrics on active Flex Gateway APIs. You can view charts related to these metrics and filter the data by date.
+* xref:flex-conn-manage-public-api.adoc[] - You can manage Flex Gateway with the API Manager API, which enables you to use the command line to communicate with API Manager.
```

Comment:
Just seeing if you meant to add text to the link here and below.

Reply:
Hey, nope all good. Antora 3 auto-resolves those titles.